### PR TITLE
LPS-174630 [Raylife AP/D2C] Adequating Premium Values to Sales Goals

### PR DESCRIPTION
**Pull request by EugenioAntunes**
[LPS-174630](https://issues.liferay.com/browse/LPS-174630)

### CODE REVIEW

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json [hunk 1]
```diff
@@ -231,8 +231,8 @@
 			"externalReferenceCode": "RAYPOL-017",
 			"policyOwnerName": "test 16",
 			"policyStatus": "expired",
-			"productExternalReferenceCode": "RAY002",
-			"productName": "Professional Liability",
+			"productExternalReferenceCode": "RAYAP-001",
+			"productName": "Auto",
 			"r_quoteToPolicies_c_raylifeQuoteId": "[$RaylifeApplication#RAYAPP-004$]",
 			"r_userToPolicies_userId": "20125",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json:237]: Please use _r_userToPolicies_userERC_

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json [hunk 2]
```diff
@@ -217,8 +217,8 @@
 			"externalReferenceCode": "RAYPOL-016",
 			"policyOwnerName": "test 15",
 			"policyStatus": "expired",
-			"productExternalReferenceCode": "RAY002",
-			"productName": "Professional Liability",
+			"productExternalReferenceCode": "RAYAP-001",
+			"productName": "Auto",
 			"r_quoteToPolicies_c_raylifeQuoteId": "[$RaylifeApplication#RAYAPP-003$]",
 			"r_userToPolicies_userId": "20125",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json:223]: Please use r_userToPolicies_userERC

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json [hunk 3]
```diff
@@ -301,8 +301,8 @@
 			"externalReferenceCode": "RAYPOL-024",
 			"policyOwnerName": "test 23",
 			"policyStatus": "expired",
-			"productExternalReferenceCode": "RAY003",
-			"productName": "Workers Compensation",
+			"productExternalReferenceCode": "RAYAP-001",
+			"productName": "Auto",
 			"r_quoteToPolicies_c_raylifeQuoteId": "[$RaylifeApplication#RAYAPP-005$]",
 			"r_userToPolicies_userId": "20125",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json:307]: Please use r_userToPolicies_userERC

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json [hunk 4]
```diff
@@ -315,7 +315,7 @@
 			"externalReferenceCode": "RAYPOL-025",
 			"policyOwnerName": "test 23",
 			"policyStatus": "declined",
-			"productExternalReferenceCode": "RAY003",
+			"productExternalReferenceCode": "RAYAP-001",
 			"productName": "Workers Compensation",
 			"r_quoteToPolicies_c_raylifeQuoteId": "[$RaylifeApplication#RAYAPP-005$]",
 			"r_userToPolicies_userId": "20125",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/3_policy.json:321]: Same here

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 5]
```diff
@@ -380,35 +380,35 @@
...
-			"r_userToSalesGoals_userId": 20125
+			"r_userToSalesGoals_userERC": "RAYUSER-001"
 		},
 		{
 			"finalReferenceDate": "2022-09-30T00:00:00Z",
-			"goalValue": "16000",
+			"goalValue": 156970,
 			"initialReferenceDate": "2022-09-01T00:00:00Z",
 			"productExternalReferenceCode": "RAYAP-001",
 			"r_userToSalesGoals_userId": 20125
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:400]: Please use _r_userToSalesGoals_userERC_

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 6]
```diff
@@ -380,35 +380,35 @@
...
-			"r_userToSalesGoals_userId": 20125
+			"r_userToSalesGoals_userERC": "RAYUSER-001"
 		},
 		{
 			"finalReferenceDate": "2022-11-30T00:00:00Z",
-			"goalValue": "16000",
+			"goalValue": 16000,
 			"initialReferenceDate": "2022-11-01T00:00:00",
 			"productExternalReferenceCode": "RAYAP-001",
 			"r_userToSalesGoals_userId": 20125
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:414]: Please use _r_userToSalesGoals_userERC_

**Review by joanagrego — CHANGES_REQUESTED** [review]
Good job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 5 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:400]: Done

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 6 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:414]: Done

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Approved! Thank you @EugenioAntunes :white_check_mark:

**Review by claricebrasil — APPROVED** [review]

**Pull request closed without merging**